### ISSUE TITLE
[Accessibility][Keyboard Navigation-Flow Chart]: Main landmark is not defined at Home Page.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2019]
**User Experience** 
Users who rely on AT will face difficulty to quickly navigate through the webpage if landmarks are not defined properly.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, turn on the narrator and navigate with 'D' i.e. shortcut for landmarks.
4. Observe the issue.

**Actual:** 
No main landmark is defined on the "Design Flow" page.

**Expected:**
Main landmark should be defined for each and every page.

**Note:**
Same issue observed at other pages i.e. Home, Design Flow, User Input, Bot Response, Notification, Settings, About.

**MAS Impacted:** [MAS2.4.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d})

**Attachment:**
[MAS2.4.1_No main landmark is defined.zip](https://github.com/microsoft/BotFramework-Composer/files/4224709/MAS2.4.1_No.main.landmark.is.defined.zip)

**Comment by ashish315** [comment]
As checked the issue is resolved, so HCL is closing the issue.
